### DISCUSSION TITLE
Error sending to invalid address

### DISCUSSION THREAD

**Discussion by yellowmatter** [Extended-Thunder/send-later#359] [Q&A]
I just sent an e-mail to two invalid addresses - because when copying, the addresses were sent with a ' in the beggining and in the end. ex:. 'johnsmith@email.com' - as a result an error began to appear from time to time informing me it could not be sent, but the e-mail is nowhere to be found, neither in the sent nor the draft boxes. Is this normal? How can i stop de add-on from sending me the same error repeatedly? I quit and opened thunderbird and it seems to fix it, but i still cannot find where the e-mail went.

**Comment by jph-sendlater** [comment]
Check your Outbox folder. It should be called `Local Folders / Outbox` or `Local Folders / Unsent Mail` or something like that.

**Reply by yellowmatter** [reply]
Checked the outbox folder on local folders but nothing was in it. Proceeded to delete the Unsent Messages and Unsent messages.msf files on the folder but this only deleted the outbox folder and now i can't send emails because the add-on tried to copy the email to the outbox before sending, as it  should, right? Restored the unsent messages files from the recycling bin and the outbox folder reappeared.
